src/content/reference/platform-api/crd/awsclusters.infrastructure.giantswarm.io.md

```diff
@@ -0,0 +1,1684 @@
+---
```

Comment:
Not relevant for CAPI because we use upstream CRDs. We also bundle them into charts, so having a definition of the upstream CRDs in our docs doesn't make sense to me either because we want to showcase as few vendor-lockin types as possible.

[repeated, please check the others as well]

Reply:
I removed awsclusters.infrastructure.giantswarm.io. Will need your help to detect other obsolete ones.